tests/test_ios_testflight_main_push_filter.py

```
@@ -82,34 +106,245 @@ def mapping_keys(text: str, indent: int) -> tuple[str, ...]:
+    write() {{
+      return this;
+    }},
+  }},
+}};
+async function runDecision() {{
+{decision_script}
+}}
+await runDecision();
+process.stdout.write(JSON.stringify({{ outputs, compareCalls, warnings }}));
+"""
+    result = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)